user-guide/Troubleshooting/Known_issues/KI_user_locked_out_after_single_failed_attempt.md

```diff
@@ -0,0 +1,33 @@
+---
+uid: KI_user_locked_out_after_single_failed_attempt
+---
+
+# User locked out after single login attempt
+
+## Affected versions
+
+From DataMiner 9.5.0 [CU12]/9.6.3 onwards.<!-- RN 20426 -->
+
+## Cause
+
+A failed login attempt will result in an additional attempt by SLNet in order to check if the password was expired. In addition, Cube will automatically and silently try to log in at startup with saved credentials, which will also lead to four additional login attempts in case the password had been changed. Cube will also try to automatically reconnect every second in the background under certain conditions. As a result of this, if the security policy only allows a small number of login attempts (e.g. 5), one failed attempt can already cause a user to be locked out.
```

Comment:
> A failed login attempt **with a Windows user** ...

I would clarify that this only applies to Windows users (local or domain), not when using any form of external authentication (SAML, RADIUS, CROWD).

Reply:
I've added a sentence below this paragraph to clarify this.